packages/ui/src/tailwind.config.ts

```diff
@@ -111,6 +114,236 @@ const config = {
+          verticalAlign: 'baseline',
+        },
+        sub: { bottom: '-0.25em' },
+        sup: { top: '-0.5em' },
+        table: {
+          textIndent: '0',
+          borderColor: 'inherit',
+          borderCollapse: 'collapse',
+        },
+        'button, input, optgroup, select, textarea': {
+          fontFamily: 'inherit',
+          fontFeatureSettings: 'inherit',
+          fontVariationSettings: 'inherit',
+          fontSize: '100%',
+          fontWeight: 'inherit',
+          lineHeight: 'inherit',
+          letterSpacing: 'inherit',
+          color: 'inherit',
+          margin: '0',
+          padding: '0',
+        },
+        'button, select': { textTransform: 'none' },
+        "button, input:where([type='button']), input:where([type='reset']), input:where([type='submit'])": {
+          WebkitAppearance: 'button',
+          backgroundColor: 'transparent',
+          backgroundImage: 'none',
+        },
+        ':-moz-focusring': { outline: 'auto' },
+        ':-moz-ui-invalid': { boxShadow: 'none' },
+        progress: { verticalAlign: 'baseline' },
+        '::-webkit-inner-spin-button, ::-webkit-outer-spin-button': {
+          height: 'auto',
+        },
+        "[type='search']": { WebkitAppearance: 'textfield', outlineOffset: '-2px' },
+        '::-webkit-search-decoration': { WebkitAppearance: 'none' },
+        '::-webkit-file-upload-button': {
+          WebkitAppearance: 'button',
+          font: 'inherit',
+        },
+        summary: { display: 'list-item' },
+        'blockquote, dl, dd, h1, h2, h3, h4, h5, h6, hr, figure, p, pre': {
+          margin: '0',
+        },
+        fieldset: { margin: '0', padding: '0' },
+        legend: { padding: '0' },
+        'ol, ul, menu': { listStyle: 'none', margin: '0', padding: '0' },
+        dialog: { padding: '0' },
+        textarea: { resize: 'vertical' },
+        'input::placeholder, textarea::placeholder': {
+          opacity: '1',
+          color: theme('colors.gray.400', '#9ca3af'),
+        },
+        'button, [role="button"]': { cursor: 'pointer' },
+        ':disabled': { cursor: 'default' },
+        'img, svg, video, canvas, audio, iframe, embed, object': {
+          display: 'block',
+          verticalAlign: 'middle',
+        },
+        'img, video': { maxWidth: '100%', height: 'auto' },
+        '[hidden]': { display: 'none' },
+      });
+
+      /**
+       * 3. Apply default theme variables (unscoped)
+       */
+      addBase({
+        ':where(:root)': {
+          '--cl-font-family':
+            '-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica,\n    Cantarell, Ubuntu, roboto, noto, arial, sans-serif',
+          '--cl-color-danger': '0 84% 60%',
+          '--cl-color-success': '142 71% 45%',
+          '--cl-color-warning': '25 95% 53%',
+          '--cl-radius': '0.375rem',
+          '--cl-spacing-unit': '1rem',
+          '--cl-font-size': '0.8125rem',
```

Comment:
Note for future PR to add all of the default color variables here.

Reply:
Following up